### DISCUSSION TITLE
Balance curent sensor

### DISCUSSION THREAD

**Discussion by twty** [syssi/esphome-jk-bms#183] [Ideas]
Hi.
i'll like a balance current sensor. Is not very util but look fine a guge that move from left to right .
Can be implemented ?
Thanks

**Comment by syssi** [comment]
The `balance current` sensor is available via BLE (`JK02`). The measurement is missing at the UART-TTL response. This cannot be fixed/changed.

**Reply by twty** [reply]
Thanks for fast reply, I prefere UART. Any way great project
